libs/sdk-ui-kit/src/Dialog/ConversionTouchPointDialogBase.tsx

```diff
@@ -0,0 +1,188 @@
+// (C) 2021 GoodData Corporation
+import React, { useEffect, useState } from "react";
+import { useIntl } from "react-intl";
+import { useHubspotForm } from "@aaronhayes/react-use-hubspot-form";
+import { HubSpotFormLocale } from "@aaronhayes/react-use-hubspot-form/build/main/hubspot-form-locale.enum";
+import { LoadingComponent } from "@gooddata/sdk-ui";
+
+import { Button } from "../Button";
+
+declare global {
+    interface Window {
+        jQuery: object;
+        hbspt: undefined | { isSuccessMessageShow: boolean };
+    }
+}
+
+/**
+ * @internal
+ */
+export interface IHubspotFormField {
+    /**
+     * Form Field Name
+     */
+    name: string;
+
+    /**
+     * Form Field Value
+     */
+    value: string | any;
+
+    /**
+     * Click action
+     */
+    click: () => void;
+}
+
+/**
+ * @public
+ */
+export interface IHubspotFormValue {
+    [key: string]: string | number | boolean;
+}
+
+/**
+ * @public
+ */
+export interface IConversionTouchPointDialogBaseProps {
+    /**
+     * Hubspot Portal Id
+     */
+    hubspotPortalId: string;
+
+    /**
+     * Hubspot Form Id
+     */
+    hubspotFormId: string;
+
+    /**
+     * Dialog Title
+     */
+    dialogTitle?: string;
+
+    /**
+     * The value for Cancel button
+     */
+    cancelButtonText?: string;
+
+    /**
+     * Show/Hide Cancel button
+     */
+    showCancelButton?: boolean;
+
+    /**
+     * Populate values for the Hubspot form
+     */
+    values?: IHubspotFormValue;
+
+    /**
+     * Mark the checkbox as checked in the Hubspot form base on this value.
+     */
+    selectedValue?: string;
+
+    /**
+     * The id for html element render the Hubspot form
+     */
+    targetId?: string;
+
+    /**
+     * Close dialog action
+     */
+    onClose: () => void;
+
+    /**
+     * Form submitted callback function
+     */
+    onFormSubmitted?: () => void;
+}
+
+/**
+ * @internal
+ */
+export const ConversionTouchPointDialogBase: React.FC<IConversionTouchPointDialogBaseProps> = (props) => {
```

Comment:
Should we rename this component `ConversionTouchPointDialog` → `HubspotConversionTouchPointDialog`, as the implementation of the component is tightly coupled with Hubspot?
※ I mean, to make it clearer for customers who don't use Hubspot that they shouldn't reuse this component.

Reply:
I have renamed them. Thanks!